script/testing/util/db_server.py

```diff
@@ -108,6 +109,46 @@ def print_db_logs(self):
         print_pipe(self.db_process)
         LOG.info("************* DB Logs End *************")
 
+    def execute(self, sql, autocommit=True, expect_result=True, user=DEFAULT_DB_USER):
+        """
+        Opens up a connection at the DB, and execute a SQL.
+
+        WARNING: this is a really simple (and barely thought-through) client execution interface. Users might need to
+        extend this with more arguments or error checking. Only SET SQl command has been tested.
+
+        :param sql: SQL to be execute
+        :param autocommit: If the connection should be set to autocommit. For SQL that should not run in transactions,
+            this should be set to True, e.g. SET XXX
+        :param expect_result: True if results rows are fetched and returned
+        :param user: User of this connection
+        :return: None if error or not expecting results, rows fetched when expect_result is True
+        """
+        try:
+            with psql.connect(port=self.db_port, host=self.db_host, user=user) as conn:
+                conn.set_session(autocommit=autocommit)
+                with conn.cursor() as cursor:
+                    cursor.execute(sql)
+
+                    if expect_result:
+                        rows = cursor.fetchall()
+                        return rows
+
+                    return None
+        except Exception as e:
+            LOG.error(f"Executing SQL = {sql} failed: ")
+            # Re-raise this
+            raise e
+
+    def turn_on_metrics_trace(self):
```

Comment:
You might want to name this `enable_metrics_trace` just to keep the terminology similar.